projects/sushiswap-furo/furo.js

```diff
@@ -0,0 +1,101 @@
+const sdk = require("@defillama/sdk");
+const { request, gql } = require("graphql-request");
+const { getChainTransform } = require("../helper/portedTokens");
+const { nullAddress } = require('../helper/unwrapLPs')
+
+const graphUrls = {
+  ethereum:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-ethereum",
+  polygon:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-polygon",
+  fantom: "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-fantom",
+  bsc: "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-bsc",
+  avax: "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-avalanche",
+  arbitrum:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-arbitrum",
+  optimism:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-optimism",
+  xdai: "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-gnosis",
+  harmony:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-harmony",
+  moonbeam:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-moonbeam",
+  moonriver:
+    "https://api.thegraph.com/subgraphs/name/sushi-subgraphs/furo-moonriver",
+};
+
+const bentoboxes = {
+  ethereum: "0xF5BCE5077908a1b7370B9ae04AdC565EBd643966",
+  polygon: "0x0319000133d3AdA02600f0875d2cf03D442C3367",
+  fantom: "0xF5BCE5077908a1b7370B9ae04AdC565EBd643966",
+  bsc: "0xF5BCE5077908a1b7370B9ae04AdC565EBd643966",
+  avax: "0x0711b6026068f736bae6b213031fce978d48e026",
+  arbitrum: "0x74c764D41B77DBbb4fe771daB1939B00b146894A",
+  optimism: "0xc35DADB65012eC5796536bD9864eD8773aBc74C4",
+  xdai: "0xE2d7F5dd869Fc7c126D21b13a9080e75a4bDb324",
+  harmony: "0xA28cfF72b04f83A7E3f912e6ad34d5537708a2C2",
+  moonbeam: "0x80C7DD17B01855a6D2347444a0FCC36136a314de",
+  moonriver: "0x145d82bCa93cCa2AE057D1c6f26245d1b9522E6F",
+};
+
+const furoQuery = gql`
+  query get_tokens($block: Int) {
+    tokens(
```

Comment:
Also I forgot to add `block: { number: $block } ` to the query, might want to add it to get historical TVL.